python/scripts/meta.json

```diff
@@ -2084,5 +2084,54 @@
             }
         ]
 
+    },
+
+    "reac_scores": {
+        "cleaner": "reac_score_cleaner",
+        "replace_table": true,
+        "fields": [
+            {
+                "display_name": "nlihc_id",
```

Comment:
It would be good to edit the display name and display text etc. It looks like you've used display_text for the purpose of display_name. display_name should be the title displayed on a legend or other field (REAC Score) whereas the helper text should be a sentence or so describing what it is ("REAC scores come from HUD when they assess the physical condition of HUD related multifamily projects, typically once every few years per property.").